src/index.js

```diff
@@ -16,6 +16,7 @@ const applyConfig = (config) => {
     restricted: false,
     mostUsed: true,
     sidebarTab: 1,
+    unwantedButtons: ['heading-three', 'blockquote'],
```

Comment:
You removed `heading-two` from the old hardcoded list, was that intentional? If so it should be mentioned in the release notes.

Reply:
Thinking it better, let's the introduction block not have any heading button. We will customize later in light-theme or per client.

Reply:
Done.